schema/block.proto

```diff
@@ -3,35 +3,44 @@ package iroha.protocol;
 import "commands.proto";
 import "primitive.proto";
 
+message Header {
+  uint64             created_time    = 1;
```

Comment:
Can we use `google.protobuf.Timestamp`?
https://github.com/google/protobuf/issues/991